init.lua

```diff
@@ -279,132 +353,154 @@ awards.register_achievement("award_marchand_de_sable", {
+		type = "dig",
+		node = "default:dry_grass_3",
+		target = 1
 	}
 })
 
--- You Suck!
-awards.register_achievement("award_you_suck", {
-	title = S("You Suck!"),
-	description = S("Die 100 times."),
+awards.register_achievement("awards_cactus", {
+	title = S("Desert Discoverer"),
+	description = S("Mine your first cactus."),
+	icon = "default_cactus_side.png",
 	trigger = {
-		type = "death",
-		target = 100
-	},
-	secret = true
-})
-
--- Burned to death
-awards.register_achievement("award_burn", {
-	title = S("You're a witch!"),
-	description = S("Burn to death in a fire.")
-})
-awards.register_onDeath(function(player,data)
-	local pos = player:getpos()
-	if pos and minetest.find_node_near(pos, 2, "fire:basic_flame") ~= nil then
-		return "award_burn"
-	end
-	return nil
-end)
-
--- Died in flowing lava
-awards.register_achievement("award_in_the_flow", {
-	title = S("In the Flow"),
-	description = S("Die in flowing lava.")
-})
-awards.register_onDeath(function(player,data)
-	local pos = player:getpos()
-	if pos and minetest.find_node_near(pos, 2, "default:lava_flowing") ~= nil then
-		return "award_in_the_flow"
-	end
-	return nil
-end)
-
--- Die near diamond ore
-awards.register_achievement("award_this_is_sad", {
-	title = S("This is Sad"),
-	description = S("Die near diamond ore.")
-})
-awards.register_on_death(function(player,data)
-	local pos = player:getpos()
-	if pos and minetest.find_node_near(pos, 5, "default:stone_with_diamond") ~= nil then
-		return "award_this_is_sad"
-	end
-	return nil
-end)
-
--- Die near diamond ore
-awards.register_achievement("award_the_stack", {
-	title = S("The Stack"),
-	description = S("Die near bones.")
-})
-awards.register_on_death(function(player,data)
-	local pos = player:getpos()
-	if pos and minetest.find_node_near(pos, 5, "bones:bones") ~= nil then
-		return "award_the_stack"
-	end
-	return nil
-end)
-
--- Die near diamond ore
-awards.register_achievement("award_deep_down", {
-	title = S("Deep Down"),
-	description = S("Die below -10000"),
-	secret = true
-})
-awards.register_on_death(function(player,data)
-	local pos = player:getpos()
-	if pos and pos.y < -10000 then
-		return "award_deep_down"
-	end
-	return nil
-end)
-
--- Die near diamond ore
-awards.register_achievement("award_no_screen", {
-	title = S("In space, no one can hear you scream"),
-	description = S("Die above 10000"),
-	secret = true
-})
-awards.register_on_death(function(player,data)
-	local pos = player:getpos()
-	if pos and pos.y > 10000 then
-		return "award_no_screen"
-	end
-	return nil
-end)
+		type = "dig",
+		node = "default:cactus",
+		target = 1
+	}
+})
+
+awards.register_achievement("awards_dry_shrub", {
+	title = S("Far Lands"),
+	description = S("Mine your first dry shrub."),
+	icon = "default_dry_shrub.png",
+	trigger = {
+		type = "dig",
+		node = "default:dry_shrub",
+		target = 1
+	}
+})
+
+awards.register_achievement("awards_farmer", {
+	title = S("Farmer"),
+	description = S("Dig a fully grown wheat plant."),
```

Comment:
Harvest sounds better. It's slightly more vague (like saying "harvest" rather than "pull out of the ground") but that shouldn't be a problem, as this isn't a tutorial mod.